### ISSUE TITLE
Problem with tag "wcag22aa"

### ISSUE THREAD

**Issue by git288hub** [dequelabs/axe-core#4673]
### Product

axe-core

### Product Version

4.10.2

### Latest Version

- [x] I have tested the issue with the latest version of the product

#### Expectation

I just want to enable 'wcag22aa' tag to run tests on my website. In documentation I saw below "WCAG 2.2 Level A & AA Rules" info: "These rules are disabled by default, until WCAG 2.2 is more widely adopted and required."

I executing axe-core.run() via:

&{results}=    Run Accessibility Tests    result_file=${path}/result_${view}.json    context=${contextLoc}    options=${optionsSpec}

in variable ${optionsSpec} = {rules: [{ tags: ['wcag22aa'], enabled: true }]}
or I also tested ${optionsSpec} = {runOnly: {type: 'tag', values: ['wcag22aa']}}

I use axe-core with Robot Framework with libraries: 
1. robotframework-axelibrary==0.1.5
2. axe-selenium-python==2.1.6 (it uses axe-core@4.10.2)

#### Actual
I get error on firefox:
Exception has occurred.
Suspended due to logged failure: TimeoutException: Message: Timed out after 5000 ms

using chrome:
Exception has occurred.
Suspended due to logged failure: TimeoutException: Message: script timeout
  (Session info: chrome=132.0.6834.84)

#### Additional context
I also tested other tags like a: {"runOnly": {"type": "tag", "values": ["wcag21aa", "wcag2a"]}} and everything works fine (in 11 different tests with different context locators), but when I'm trying to add "wcag22aa" I got error. In my tests I use selenium robotframework-seleniumlibrary==6.5.0 and when I extended timeout to 30s the error also occured.

**Comment by straker** [comment]
Thanks for the issue. Just so you're aware, we don't maintain or own either `axe-selenium-python` or `robotframework-axelibrary`, so you may need to file the bug with them. When I run `await axe.run({runOnly: {type: 'tag', values: ['wcag22aa']}})` I don't get any errors.

That being said, would it be possible to provide a link to the site where the issue is occurring? We'd like to make sure that running the `target-size` rule on the site isn't causing a problem.

**Comment by WilcoFiers** [comment]
Closing this since there was no response. Feel free to reopen if necessary.
